src/common/components/ui/ClusterEventsToolbar.tsx

```diff
@@ -177,9 +174,9 @@ const ClusterEventsToolbar: React.FC<ClustersListToolbarProps> = ({
     });
   };
 
-  const onDeleteChip: ToolbarFilterProps['deleteChip'] = (type, chip) => {
+  const onDeleteChip: ToolbarFilterProps['deleteChip'] = (type, chip: ToolbarChip | string) => {
     if (type) {
-      const id = chip['key'] || chip;
+      const id = (chip['key'] || chip || '') as string;
```

Comment:
BTW, you can use `??` instead of `||`. The former will use the RHS when the LHS is either `null` or `undefined`. It doesn't coerce the LHS into a boolean.